### DISCUSSION TITLE
Next JS <Link requires double tapping on mobile

### DISCUSSION THREAD

**Discussion by vivbarajas** [vercel/next.js#60983] [Help]
### Summary

On mobile devices, I have to double tap the the link in order for the link to route correctly. Here is the code:

```
 <Link
          href="/methodology"
          passHref={true}
          target="_blank"
          rel="noopener noreferrer"
          className="w-auto text-right text-sm text-riskfactor-400 underline underline-offset-2 hover:text-riskfactor-300 md:w-[345px]"
          onClick={() =>
            handleLinkAnalytics(
              'How is is it calculated?',
            ,
            )
          }
        >
          How is this Calculated?
        </Link>
```

**Comment by fredrik-sogaard** [comment]
I'm experiencing this too. I found that adding `prefetch=false` fixed the issue in our case.

**Comment by roosemberth** [comment]
We're also having this issue, but only on iPhone.

**Comment by vincent38wargnier** [comment]
Same issue, prefetch does't change anything

**Comment by badrabkar** [comment]
did you solve the problem ??

**Comment by imceZZ** [comment]
any solution for this I also tried `prefetch=false` but doesn't work

**Comment by vivbarajas** [comment]
Hi Neim,

Thank you for your message. I haven't seen any updates on this issue
either, and it still seems to be persisting for me as well, even with
`prefetch=false`.

Viviana


On Mon, Aug 18, 2025 at 12:12 PM Neim Elezi ***@***.***>
wrote:

> any solution for this I also tried prefetch=false but doesn't work
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/next.js/discussions/60983#discussioncomment-14143248>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIUJKNTIYUGG4RITUOVWE6T3OH3PFAVCNFSM6AAAAABV5WJ67WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTIMJUGMZDIOA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
[image: First Street] <https://firststreet.org>
*Viviana Barajas*
Software Engineer
***@***.***

**Reply by imceZZ** [reply]
@vivbarajas  remove link try to use onTouchStart={() => router.push()}  works much better on mobile devices
